mentors/cpp/g.json

```
@@ -7,4 +7,12 @@
     "avatar_url": null,
     "bio": "I have used C++ at work for 15 years. I aspire for performance in my coding."
   }
```

```suggestion
  },
```
A comma is needed here.